src/Sarif.UnitTests/Visitors/ReformattingVisitorTests.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Copyright (c) Microsoft. All rights reserved. Licensed under the MIT        
+// license. See LICENSE file in the project root for full license information. 
+
+using System;
+using FluentAssertions;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Sarif.Visitors
+{
+    public class ReformattingVisitorTests
+    {
+        //                                  0          10          20 
+        //                                  01234567 8 901234 5 678901 2 345
+        private static string s_testText = "line1\r\nline2\r\nline3\r\nline4";
+
+        private static Tuple<Region, string>[] s_testCases = new Tuple<Region, string>[]
```

Comment:
`private static readonly`

Reply:
what's the point of making the array reference read only? when i can arbitrarily swap contents of the array. swtiched to readonlycollection, though I'm not sure i understand what the additional work is buying us. :)

---
In reply to: [202095850](https://github.com/Microsoft/sarif-sdk/pull/942#discussion_r202095850) [](ancestors = 202095850)

Reply:
For me it's a matter of habit. I _always_ mark data I don't want changed. I'm a creature of habit; if I always do it, I won't forget to do it when it matters. YMMV.

Reply:
Persuasive and I did it. Also useful to build the internal muscle of recalling the pattern to author.

---
In reply to: [202413355](https://github.com/Microsoft/sarif-sdk/pull/942#discussion_r202413355) [](ancestors = 202413355)